src/jit/regset.h

```diff
@@ -411,32 +414,44 @@ struct RegValDsc
         unsigned rvdLclVarNum; // for rvdKind == RV_LCL_VAR, RV_LCL_VAR_LNG_LO, RV_LCL_VAR_LNG_HI
     };
 };
+#endif // LEGACY_BACKEND
 
 class RegTracker
 {
     Compiler* compiler;
     RegSet*   regSet;
+#ifdef LEGACY_BACKEND
     RegValDsc rsRegValues[REG_COUNT];
+#endif
 
 public:
     void rsTrackInit(Compiler* comp, RegSet* rs)
     {
         compiler = comp;
         regSet   = rs;
+#ifdef LEGACY_BACKEND
         rsTrackRegClr();
+#endif
     }
 
+#ifdef LEGACY_BACKEND
```

Comment:
All these function declaration could be sorted out to minimize the number of ifdefs. I skipped sorting to make the diff easier to read.